AutoDuck/common_top.mak

```
@@ -1,13 +1,12 @@
 # Common AutoDuck make file
 
-AD		  = bin\autoduck.exe /Spythonwin.fmt
+AD        = bin\autoduck.exe /Spythonwin.fmt
 ADHTMLFMT = bin\autoduck.exe /Spyhtml.fmt
 ADHTMLLOG = /R html "/L$(GENDIR)\$(TARGET).HHLOG" /N
-ADLOG	  = /R help "/L$(GENDIR)\$(TARGET).HLOG" /N
-ADHTML	  = /R html "/G$(GENDIR)\$(TARGET).DUMP" "/C$(GENDIR)\$(TARGET).HHLOG" "/I$(GENDIR)\$(TARGET).IDX" "/O$(GENDIR)\$(TARGET).HTML" /D "title=$(TITLE)"
-ADHLP	  = /R help "/C$(GENDIR)\$(TARGET).HLOG" "/O$(GENDIR)\$(TARGET).RTF" /D "title=$(TITLE)"
-ADDOC	  = /RD "/O$(GENDIR)\$(TARGET).DOC" /D "doc_header=$(DOCHDR)"
-ADTAB	  = 8
-HC		  = hcw /a /c /e
-HHC		  = hhc
-PYTHON   = py -2.7
+ADLOG     = /R help "/L$(GENDIR)\$(TARGET).HLOG" /N
+ADHTML    = /R html "/G$(GENDIR)\$(TARGET).DUMP" "/C$(GENDIR)\$(TARGET).HHLOG" "/I$(GENDIR)\$(TARGET).IDX" "/O$(GENDIR)\$(TARGET).HTML" /D "title=$(TITLE)"
+ADHLP     = /R help "/C$(GENDIR)\$(TARGET).HLOG" "/O$(GENDIR)\$(TARGET).RTF" /D "title=$(TITLE)"
+ADDOC     = /RD "/O$(GENDIR)\$(TARGET).DOC" /D "doc_header=$(DOCHDR)"
+ADTAB     = 8
+HC        = hcw /a /c /e
+HHC       = hhc
```

If you still want this to be runnable w/o the python script, I think this should work to add a default executable location:
```suggestion
HHC       = hhc
PYTHON    = python
```